synapse/models/infotech.py

```diff
@@ -303,6 +462,46 @@ def getBaseModels():
                     ('reg:int', {'ptype': 'it:dev:int', 'doc': 'The integer value that was deleted (parsed from :reg).', 'ro': 1}),
                     ('reg:bytes', {'ptype': 'file:bytes', 'doc': 'The binary data that was deleted (parsed from :reg).', 'ro': 1}),
                 )),
+                ('it:prod:soft', {}, (
+                    ('name', {'ptype': 'str:lwr', 'ro': 1, 'req': 1,
+                              'doc': 'Name of the software'}),
+                    ('desc', {'ptype': 'str:txt', 'doc': 'A description of the software'}),
+                    ('desc:short', {'ptype': 'str:lwr', 'doc': 'A short description of the software'}),
+                    ('author:org', {'ptype': 'ou:org', 'doc': 'Organization responsible for the software', }),
+                    ('author:acct', {'ptype': 'inet:web:acct', 'doc': 'Web user responsible for the software', }),
```

Comment:
"Responsible" org / acct / person may change - for example, freeware tools developed by one company that is later acquired by another company (e.g., Sysinternals -> Microsoft, Foundstone -> McAfee, or even Mandiant -> FireEye). This is probably an "edge case" that can be addressed "by convention" (leave original owner or update secondary props or simply reflect via org / suborg relationship).